src/components/reusable-ui/PrimaryButton.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components"
+import { theme } from "../../theme"
+
+export default function PrimaryButton({ label, Icon }) {
+  return (
+    <PrimaryButtonStyled type="submit">
```

Comment:
```suggestion
    <PrimaryButtonStyled >
```

Ici, on est dans le cadre d'un composant réutilisable qui ne va pas forcément être utilisé dans un form. Donc lui assigner d'emblée l'attribut type="submit", c'est ajouter du spécifique dans un composant réutilisable et donc générique.